packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -124,11 +127,18 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
 
     if (src) {
       this.setState({
-        loadState: ImageLoadState.loaded
+        loadState: ImageLoadState.loaded,
+        cached: true
       });
     }
   }
 
+  private _isImageCached(src: string): boolean {
```

Comment:
This looks like a performance concern. Now every time the component receives props, it's creating an img element?